python/lsst/sims/maf/metrics/calibrationMetrics.py

```diff
@@ -2,8 +2,10 @@
 from .baseMetric import BaseMetric
 from lsst.sims.maf.utils.astrometryUtils import sigma_slope, m52snr, astrom_precision
 import lsst.sims.maf.utils as utils
```

Comment:
these two import lines could be combined, to just 
import lsst.sims.maf.utils as mafUtils
and then referring to m52snr as mafUtils.m52snr
(and other associated items). This would be clearer throughout the file.